.github/workflows/multicluster.yaml

```diff
@@ -66,8 +66,10 @@ jobs:
 
       - name: Install cilium in cluster2
         run: |
-          cilium install --cluster-name ${{ env.clusterName2 }} --cluster-id 2 --kube-proxy-replacement disabled --config monitor-aggregation=none
-          cilium hubble enable
+          CONTEXT1=$(kubectl config view | grep ${{ env.clusterName1 }} | head -1 | awk '{print $2}')
```

Comment:
I merely copy/pasted these from other steps. Preferably we would make these less fragile like you showed and also define these only once somehow. I'll leave this for a follow-on PR.

Reply:
We need a substring match to find the context. This is hard to do when `kubectl` jsonpath does not support regular expressions: https://github.com/kubernetes/website/pull/23141

Reply:
Also, in this script the cluster name is specific to the test run instance, so the grep is actually very specific.

Reply:
Funnily enough, I also found this to be brittle when doing personal tests. I would recommend this `CONTEXT1="$(kubectl config get-contexts -o name | grep ${{ env.clusterName1 }})"`.

Reply:
Could also simply use `{.contexts[0].name}` and `{.contexts[1].name}` if there are only going to be 2 contexts in the config.

Reply:
That is right! I've kept the `grep` because actually this is what I'm going to put in the cluster cheatsheet, and we developers tend to have more than just these 2 contexts 😇

Reply:
Though now I do wonder if context order is stable? If not, then we should stick to `grep`.

Reply:
I adopted Nicolas' suggestion and cleaned up the code by setting the context strings to environment variables.